.github/workflows/nightly.yml

```diff
@@ -6,10 +6,8 @@ on:
     - cron: "0 4 * * *"
 
 env:
-  CI_GRADLE_ARG_PROPERTIES: >
-    -Porg.gradle.jvmargs=-Xmx4g
-    -Porg.gradle.parallel=false
-    --no-daemon
+  GRADLE_OPTS: -Dorg.gradle.jvmargs="-Xmx3072m -Dfile.encoding=UTF-8 -XX:+HeapDumpOnOutOfMemoryError" -Dkotlin.daemon.jvm.options="-Xmx2560m" -Dkotlin.incremental=false
+  CI_GRADLE_ARG_PROPERTIES: --stacktrace -PpreDexEnable=false --max-workers 2 --no-daemon
```

Comment:
I think we can remove the `--stacktrace` option on the line we call `CI_GRADLE_ARG_PROPERTIES` like you did on other config files. The `nightly_build.md` doc could be updated as well.

Reply:
great catch, will do! :+1:

Reply:
https://github.com/vector-im/element-android/pull/6728/commits/8470ac9731c18bfd12945d3afbc2722ab61d3b44